draft-kazuho-httpbis-priority.md

```diff
@@ -141,51 +141,52 @@ priority = urgency=-1
 The definition of the urgencies and their expected use-case are described below.
 Endpoints SHOULD respect the definition of the values when assigning urgencies.
 
-### blocking
+### prerequisite
 
-The blocking urgency (value -1) indicates that the response prevents other
-responses from being used.
+The prerequisite urgency (value -1) indicates that the response prevents other
+responses with an urgency of prerequisite or default from being used.
 
 For example, use of an external stylesheet can block a web browser from
-rendering the HTML. In such case, the stylesheet is given the blocking urgency.
+rendering the HTML. In such case, the stylesheet is given the prerequisite
+urgency.
 
-### document
+### default
 
-The document urgency (value 0) indicates that the response contains the document
-that is being processed.  This urgency is also used for responses that deserve
-the same precedence as the contents of the document.
+The default urgency (value 0) indicates a response that is to be used as it is
+delivered to the client, but one that does not block other responses from being
+used.
 
-For example, when a user using a web browser navigates to a different HTML
-document, the request for that HTML is given the document urgency.  When that
-HTML document uses a custom font, the request for that custom font SHOULD also
-be given the document urgency.  This is because the availablity of the custom
-font is likely a precondition for the user to use that portion of the HTML
-document, which is to be rendered by that font.
+For example, when a user using a web browser navigates to a new HTML document,
+the request for that HTML is given the default urgency.  When that HTML document
+uses a custom font, the request for that custom font SHOULD also be given the
+default urgency.  This is because the availablity of the custom font is likely
+a precondition for the user to use that portion of the HTML document, which is
+to be rendered by that font.
 
-### non-blocking
+### supplementary
 
-The non-blocking urgency indicates that the response does not prevent the client
-from using the document even though the response is being incorporated into or
-referred to by the document.
+The supplementary urgency indicates a response that is helpful to the client
+using a composition of HTTP responses, even though the response itself is not
```

Comment:
nit, you previously made an editorial pass to remove `HTTP`
```suggestion
using a composition of responses, even though the response itself is not
```

Reply:
Thanks! Applied.